test/e2e/app-dir/rsc-basic/rsc-basic.test.ts

```diff
@@ -407,7 +407,7 @@ describe('app dir - rsc basics', () => {
         const hasRCScript = /\$RC=function/.test(chunk)
         if (hasRCScript) results.push('refresh-script')
 
-        const isFallbackResolved = chunk.includes('fallback')
```

Comment:
`fallback` now matches Next.js source that was sent as debugging info. Specifically, `const fallbackParams`